src/OSPSuite.Core/Domain/Builder/SimulationBuilder.cs

```diff
@@ -92,7 +92,8 @@ private void performMerge()
          _observers.AddRange(allBuilder(x => x.Observers));
          _molecules.AddRange(allBuilder(x => x.Molecules));
          _parameterValues.AddRange(allStartValueBuilder(x => x.SelectedParameterValues));
-         _initialConditions.AddRange(allStartValueBuilder(x => x.SelectedInitialConditions));
+         _initialConditions.AddRange(allStartValueBuilder(x => x.SelectedInitialConditions)
```

Comment:
Now we need to look in all the expression profiles to build the list of all InitialConditions